spice_agent/inference/actions.py

```diff
@@ -44,6 +44,7 @@ def _update_inference_job(
         status: str,
         was_guarded: Optional[bool] = None,
         text_output: Optional[str] = None,
+        options: Optional[Dict[str, Any]] = None,
```

Comment:
not sure if you need to send the options BACK to the server, they shouldn't have changed